src/wx/wxvbam.h

```diff
@@ -500,11 +504,42 @@ class DrawingPanelBase;
 #include <windows.h>
 #endif
 
-class GameArea : public wxPanel, public HiDPIAware {
+#ifndef NO_THREAD_MAINLOOP
+#include <wx/thread.h>
+
+wxDEFINE_EVENT(WX_THREAD_REQUEST_UPDATEDRAWPANEL, wxThreadEvent);
+wxDEFINE_EVENT(WX_THREAD_REQUEST_DRAWFRAME, wxThreadEvent);
+wxDEFINE_EVENT(WX_THREAD_REQUEST_UPDATESTATUSBAR, wxThreadEvent);
+wxDEFINE_EVENT(WX_THREAD_REQUEST_GBPRINTER, wxThreadEvent);
+wxDEFINE_EVENT(WX_THREAD_REQUEST_UPDATELOG, wxThreadEvent);
+#endif // NO_THREAD_MAINLOOP
+
+class GameArea : public wxPanel, public HiDPIAware
+#ifndef NO_THREAD_MAINLOOP
+                 , public wxThreadHelper
+#endif
+{
 public:
     GameArea();
     virtual ~GameArea();
 
+#ifndef NO_THREAD_MAINLOOP
+    virtual wxThread::ExitCode Entry();
+    void StartEmulationThread();
```

Comment:
Where are these methods defined? Did you forget to include a file in your commit?

Reply:
Sorry I didn't see that some sections of the github diff view need to be expanded.

Reply:
No they are actually not there, I misunderstood the point of the expand buttons.